python/py-ipympl/Portfile

```diff
@@ -4,13 +4,15 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-ipympl
-version             0.7.0
+version             0.9.3
 revision            0
 license             BSD
 supported_archs     noarch
 platforms           {darwin any}
 
-python.versions     37 38 39 310
+python.versions     37 38 39 310 311
+python.pep517       no
+# Installation with pep517 fails due to incorrect dependency version requirements in pyproject.toml
```

Comment:
this should build with `python.pep517 yes`; I am not sure what error you're seeing but if you verified that the version in MacPorts works (which is probably the case since you that is the versions from MacPorts you are using), then you should just patch the version number in `pyproject.toml`.